src/css/components/menu/quirks.css

```diff
@@ -0,0 +1,9 @@
+/**
+ * This rule prevents the bold font to be loaded until the menu is ready,
+ * since the menu is the only place need ing this `font-weight`. It is
+ * used for the color scheme when “Auto” (shown in bold) is chosen.
+ */
+
```

Comment:
```suggestion
/**
 * This rule prevents the bold font to be loaded until the menu is ready,
 * since the menu is the only place needing this `font-weight` when it
 * displays “Auto” (in bold) when the system color scheme is chosen.
 */
```